models/observation.go

```diff
@@ -0,0 +1,82 @@
+package models
+
+// ObservationDoc represents information (metadata) relevant to a version
+type ObservationDoc struct {
+	Dimensions          map[string]Options `json:"dimensions,omitempty"`
+	Context             string             `json:"context,omitempty"`
+	Links               *ObservationLinks  `json:"links,omitempty"`
+	Observation         string             `json:"observation,omitempty"`
```

Comment:
not sure these should all be omit empty? if there's no dimensions how can we have an observation?